RecoLocalMuon/GEMRecHit/test/BuildFile.xml

```diff
@@ -27,7 +27,7 @@
 
 <use   name="MagneticField/Engine"/>
 <use   name="MagneticField/Records"/>
-
+<use   name="DQMServices/Core"/>
```

Comment:
this was previously removed by #11327
I think there should be no changes in this file in this PR